static/js/app.ts

```diff
@@ -272,12 +276,13 @@ export function initializeCodePage(options: InitializeCodePageOptions) {
 function attachMainEditorEvents(editor: HedyEditor) {
 
   editor.on('change', () => {
-    theLocalSaveWarning.setProgramLength(theGlobalEditor.contents.split('\n').length);
-    theGlobalEditor.clearIncorrectLines();
+    theLocalSaveWarning.setProgramLength(theGlobalEditor.contents.split('\n').length);    
   });
 
   // If prompt is shown and user enters text in the editor, hide the prompt.
   editor.on('change', function() {
+    // TODO: When the user clicks back in the editor when an ask is open
```

Comment:
Iiinteresting. I thought we used to have the behavior that the ask would be closed if they did that?

Reply:
It does closes, but the editor loses focus again after typing a something. I opened #4742 to remind me of this issue!